build_mi_proto.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
This script would fit better in `gui/scripts/`.

Reply:
The script puts the proto files in `dist-assets/`. Would you also want to move them to somewhere in `gui/`?

Reply:
I don’t think we have a specific directory for content like this but yeah, moving the `mi_proto` directory to `gui/scripts/mi_proto` would make it consistent with some other scripts 👍

Reply:
`build_mi_proto.sh` now lives in `gui/scripts`, and it creates its artefacts directory alongside itself (`gui/scripts/mi_proto`)